spot_driver/spot_driver/publish_cameras.py

```diff
@@ -18,6 +18,8 @@
 from spot_driver.ros_helpers import get_from_env_and_fall_back_to_param
 import rclpy.node
 
+from .ros_helpers import bosdyn_data_to_image_and_camera_info_msgs
```

Comment:
Do we have to use a relative import here?

Reply:
Didn't work when I didn't use the relative import and we use the same one in `spot_ros2.py`